tests/Api/DataFixtures/ORM/product_variant.yaml

```
@@ -0,0 +1,22 @@
+Sylius\Component\Core\Model\Product:
+    product:
+        fallbackLocale: en_US
+        currentLocale: en
+        code: MUG_SW
+
+Sylius\Component\Product\Model\ProductVariantTranslation:
+    productVariantTranslation:
```

The current convention in other files is to use snake case notation
```suggestion
    product_variant_translation:
```